src/test/java/io/fabric8/vertx/maven/plugin/it/ExtraManifestInfoIT.java

```diff
@@ -0,0 +1,223 @@
+
+        return svnurl;
+    }
+
+    @Test
+    public void testGITSCM() throws IOException, VerificationException, GitAPIException {
+        File testDir = initProject(GIT_PROJECT_ROOT);
+        assertThat(testDir).isDirectory();
+
+        initVerifier(testDir);
+
+        prepareProject(testDir, verifier);
+
+        File gitFolder = GitUtil.findGitFolder(testDir);
+
+        assertThat(testDir).isNotNull();
+        assertThat(testDir.getName()).endsWith("manifest-git-it");
+        assertThat(gitFolder).isNull();
+
+        Git git = prepareGitSCM(testDir, verifier);
+        gitFolder = git.getRepository().getDirectory();
+        assertThat(gitFolder.getParentFile().getName()).isEqualTo(testDir.getName());
+        assertThat(git.status().call().getUntracked()).contains("pom.xml", "src/main/java/demo/SimpleVerticle.java");
+
+        //Now add and commit the file
+        DirCache index = git.add().addFilepattern(".").call();
+        assertThat(index.getEntryCount()).isEqualTo(2);
+
+        git.commit().setMessage("First Import").call();
+
+        runPackage(verifier);
+        assertManifest(testDir, "git");
+
+    }
+
+    @Test
+    public void testSVNSCM() throws SVNException, IOException, VerificationException {
+        File testDir = initProject(SVN_PROJECT_ROOT);
+        assertThat(testDir).isDirectory();
+
+        initVerifier(testDir);
+
+        prepareProject(testDir, verifier);
+
+        File svnFolder = SVNUtil.findSvnFolder(testDir);
+
+        assertThat(testDir).isNotNull();
+        assertThat(testDir.getName()).endsWith("manifest-svn-it");
+        assertThat(svnFolder).isNull();
+
+        SVNURL svnurl = prepareSVNRepo(testDir, verifier);
+
+        assertThat(testDir).isNotNull();
+        assertThat(testDir.getName()).endsWith("manifest-svn-it");
+
+        //Checkout sources from SVN Repo to test dir
+        SVNClientManager svnClientManager = SVNClientManager.newInstance();
+        SVNUpdateClient svnUpdateClient = svnClientManager.getUpdateClient();
+        long checkout = svnUpdateClient.doCheckout(svnurl, testDir, SVNRevision.HEAD,
+            SVNRevision.HEAD, SVNDepth.fromRecurse(true), false);
+
+        svnFolder = SVNUtil.findSvnFolder(testDir);
+        assertThat(svnFolder).isNotNull();
+        assertThat(checkout).isEqualTo(1);
+
+        runPackage(verifier);
+        assertManifest(testDir, "svn");
+
+    }
+
+    private void assertManifest(File testDir, String scm) throws IOException {
+        verifier.assertFilePresent("target/vertx-demo-start-0.0.1.BUILD-SNAPSHOT.jar");
+        File jarFile = new File(testDir, "target/vertx-demo-start-0.0.1.BUILD-SNAPSHOT.jar");
+        assertThat(jarFile).isNotNull();
+
+        Manifest manifest = new JarFile(jarFile).getManifest();
+        //Extract and Check Manifest for details
+        assertThat(manifest).isNotNull();
+
+        //manifest.write(System.out);
```

Comment:
had plans to remove that sys out before merge, since at times i want to check the manfest out ;)